source/manual/rabbitmq.html.md

```diff
@@ -78,3 +78,19 @@ Note: there is a risk that you might delete the wrong message(s). This
 is because the contents of the queue may have changed.
 
 Repeat, but change the "Requeue" option to "No".
+
+## Previewing a message for a document_type
+
+The publishing API generates messages when content is updated and posts them
+to the rabbit MQ exchange. Each message has a shared format, however teh contents
+of the message is affected by the publoishing app and what data it sends to the
+publioshing API.
```

Comment:
There's a few typos here, can you do another edit?

- publoishing/publishging
- teh
- RabbitMQ
- to allow easily generate
- generetd
- As message for different formats can vary

I'd also explicitly name the exchange (published_documents) so people can easily track down the messages in the management UI.

Reply:
@brenetic can you pair with me on this so we can get it merged. :)